docs/api/maintenance.rst

```diff
@@ -0,0 +1,11 @@
+Maintenance Mode
+==================
+
+.. contents::
+  :local:
+
+.. autoclass:: pubtools.pulplib.MaintenanceEntry
+    :members:
+
+.. autoclass:: pubtools.pulplib.MaintenanceReport
```

Comment:
Could we please swap the order of the classes here?  MaintenanceReport doc string contains some general explanation of what maintenance mode is, so it seems to make more sense for that to come first.